android/tangram/src/main/cpp/platform_android.cpp

```diff
@@ -423,7 +398,7 @@ void markerPickCallback(jobject listener, const Tangram::MarkerPickResult* marke
 
     jniEnv->CallVoidMethod(listener, onMarkerPickMID, markerPickResultObject, position[0], position[1]);
     jniEnv->DeleteGlobalRef(listener);
-
+    jniEnv->DeleteGlobalRef(tangramInstance);
```

Comment:
Why delete a global ref for this object? Did we create a global ref to it somewhere?